index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    
+    <!----------navbar setion------------------->
+    <nav class="nav-container">
+        <a href="">John Doe</a>
+        <ul class="items-menu">
+            <a href="#">
+                <li>Portfolio</li>
+            </a>
+            <a href="#">
+                <li>About</li>
+            </a>
+            <a href="#">
+                <li>Contact</li>
+            </a>
+            <a href="#">
+                <li><i class="fa-solid fa-envelope"></i></li>
+            </a>
+
+        </ul>
+        <a href="#">
+            <li><i class="fa-solid fa-bars"></i></li>
+        </a>
+
+    </nav>
+    <!--------------Hero section------------------------------------------->
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+            </div>
+            <div class="hero-right">
+                <h1>Hi There.<br>I am Hyder</h1>
+                <h5>I'm a web Developer</h5>
+                <p>I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you
+                    see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div class="hero-right1">
+                <img src="Illustration-header.png">
+            </div>
```

Comment:
- [ ] Kindly make sure to insert the background image on the page using the CSS `background` properties. I highly recommend you implement this because the most professional and efficient way of inserting the background image styles we must use CSS `background` properties. For more info please read[ this article.](https://www.freecodecamp.org/news/html-background-image-how-to-add-wallpaper-images-to-your-website/)